internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec_test.go

```diff
@@ -1210,6 +1210,57 @@ var _ = Describe("Validate clusterlogforwarderspec", func() {
 			Expect(clfStatus.Pipelines[pipelineName]).To(HaveCondition(loggingv1.ConditionReady, false, loggingv1.ReasonInvalid, "invalid: unrecognized filters*"))
 		})
 
+		It("should fail with Loki and without URL (not default output)", func() {
+			forwarderSpec := &loggingv1.ClusterLogForwarderSpec{
+				Outputs: []loggingv1.OutputSpec{
+					{
+						Name: "foo",
+						Type: loggingv1.OutputTypeLoki,
+					},
+				},
+				Pipelines: []loggingv1.PipelineSpec{
+					{
+						InputRefs:  []string{loggingv1.InputNameApplication},
+						OutputRefs: []string{"foo"},
+						Name:       pipelineName,
+					},
+				},
+			}
+			clf := loggingv1.ClusterLogForwarder{}
+			clf.Spec = *forwarderSpec
+			clf.Name = constants.SingletonName
+			clf.Namespace = constants.OpenshiftNS
+
+			_, clfStatus = ValidateInputsOutputsPipelines(clf, client, extras)
+			Expect(clfStatus.Outputs["foo"]).To(HaveCondition(loggingv1.ConditionReady, false, loggingv1.ReasonInvalid, "URL is required for output type loki"))
+		})
+
+		It("should pass with Loki and without URL for default output", func() {
```

Comment:
This test should not be required or rewritten to confirm default loki does require URL